prs/PizzaStore.ts

```diff
@@ -0,0 +1,40 @@
+class PizzaStore {
+    public createPizza(style: string, type: string): Pizza | null {
+        let pizza: Pizza = undefined;
+
+        if (style === "NY") {
+            if (type === "cheese") {
+                pizza = new NYStyleCheesePizza();
+            } else if (type === "veggie") {
+                pizza = new NYStyleVeggiePizza();
```

Comment:
NYStyleVeggiePizza need implementation / make it a cheese pizza and add veggies to it.

Reply:
Same for next two.